.travis.yml

```diff
@@ -230,6 +230,27 @@ matrix:
       arch: amd64
       env:
         - TARGET=x86_64-apple-darwin
+    
+    # WebAssembly (wasm-bindgen)
+
+    - name: "wasm32-unknown-unknown (node)"
+      os: linux
+      arch: amd64
+      install:
+        - curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+      script:
+        - wasm-pack test --node --headless
+    
+    - name: "wasm32-unknown-unknown+simd128 (chrome)"
+      os: linux
+      arch: amd64
+      addons:
+        chrome : stable
+      install:
+        - curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+      script:
+        - RUSTFLAGS="-C target-feature=+simd128"
+        - wasm-pack test --chrome --headless
```

Comment:
Would it make sense to also test in firefox?

Reply:
`wasm-pack` seems to be able to test in multiple environments using a single invocation. 

https://rustwasm.github.io/docs/wasm-pack/commands/test.html?highlight=firefox#test-environment

> `wasm-pack test --node --firefox --chrome --safari --headless`

Reply:
Do we know if Firefox has SIMD support? I'm testing in Chrome here because it has experimental support with `--enable-features=WebAssemblySimd`.

Reply:
We could technically test in Node too, but I couldn't figure out how to get command-line Node args passed through `wasm-pack test`

Reply:
I went through this process a while ago, I wasn't able to enable SIMD in node or Firefox either, just chrome.  If there is a way to pass node flags it's definitely not obvious.